react/features/overlay/components/AbstractReloadableOverlay.js

```diff
@@ -0,0 +1,20 @@
+/* global APP */
+import AbstractOverlay from './AbstractOverlay';
+
+/**
+ * Implements an abstract React Component for overlay with ability to
+ * reload the page.
+ */
+export default class AbstractReloadableOverlay extends AbstractOverlay {
```

Comment:
While that is true, it forces us to add an unnecessary abstraction for the purpose of charing one line of code...